packages/eslint/src/executors/lint/lint.impl.ts

```diff
@@ -51,12 +51,6 @@ export default async function run(
     joinPathFragments(workspaceRoot, 'eslint.config.js')
   );
 
-  if (!eslintConfigPath && hasFlatConfig) {
-    const projectRoot =
-      context.projectsConfigurations.projects[context.projectName].root;
-    eslintConfigPath = joinPathFragments(projectRoot, 'eslint.config.js');
```

Comment:
Just checking - will `FlatESLint` resolve the closest `eslint.config.js` file automatically if we don't pass it explicitly (like the original ESLint class did)? Do we have an e2e case that covers this removal/flat config in general?

Reply:
It's a valid question. I tested it on an isolated repo, but I'll add an E2E test to have this officially verified.